elements/pfe-modal/src/pfe-modal.html

```diff
@@ -0,0 +1,14 @@
+<slot name="pfe-modal--trigger"></slot>
+<section class="pfe-modal__overlay" hidden>
```

Comment:
@castastrophe @starryeyez024 I applied the class name `pfe-modal__overlay` which is technically correct since it contains the overlay styles.